src/components/card/card.scss

```diff
@@ -77,6 +40,43 @@
             .col-sm-9 {
                 position: relative;
             }
+
+            .validation-message {
+                $arrow-border-width: 1rem;
+                display: block;
+                position: absolute;
+                top: 0;
+                left: 0;
+                transform: translate(16rem, 0);
+                margin-left: $arrow-border-width;
+                border: 1px solid $active-gray;
+                border-radius: 5px;
+                background-color: $ui-orange;
+                padding: 1rem;
+                max-width: 18.75rem;
+                min-height: 1rem;
+                overflow: visible;
+                color: $type-white;
+
+                &:before {
+                    display: block;
+                    position: absolute;
+                    top: 1rem;
+                    left: -$arrow-border-width / 2;
+
+                    transform: rotate(45deg);
+
+                    border-bottom: 1px solid $active-gray;
+                    border-left: 1px solid $active-gray;
+                    border-radius: 5px;
+
+                    background-color: $ui-orange;
+                    width: $arrow-border-width;
+                    height: $arrow-border-width;
+
+                    content: "";
+                }
+            }
```

Comment:
What's the reason for nesting this under `.form`? You should use as few levels of nesting as possible (but no less) because then your styles apply in more situations without having to redo the CSS.  So here, unless the validation message _shouldn't_ look this way outside of a `.form`, then it shouldn't be specific to `.form`.  Since this is imported by the Card component, then it should be scoped to `.card`, however.